fdbserver/DataDistribution.actor.cpp

```diff
@@ -463,6 +464,35 @@ struct DataDistributor : NonCopyable, ReferenceCounted<DataDistributor> {
 				    .detail("DestID", iShard.destId);
 			}
 
+			if (CLIENT_KNOBS->DD_PHYSICAL_SHARD_CORE) {
+				std::vector<ShardsAffectedByTeamFailure::Team> destTeams;
+				destTeams.push_back(ShardsAffectedByTeamFailure::Team(iShard.primaryDest, true));
+				if (self->configuration.usableRegions > 1) {
+					destTeams.push_back(ShardsAffectedByTeamFailure::Team(iShard.remoteDest, false));
+				}
+				if (iShard.srcId != anonymousShardId) {
+					self->physicalShardCollection->updatePhysicalShardToTeams(
+					    iShard.srcId.first(), teams, self->configuration.storageTeamSize, 0);
+					if (self->physicalShardCollection->physicalShardCollection.count(iShard.srcId.first()) == 0) {
+						self->physicalShardCollection->physicalShardCollection.insert(std::make_pair(
+						    iShard.srcId.first(), PhysicalShardCollection::PhysicalShard(iShard.srcId.first())));
+					}
+				}
```

Comment:
nit: convert this code block to  `PhysicalShardCollection::addPhysicalShard(id, teams)`?

Reply:
Changed. Thanks!